framework/archiver/tasks.py

```diff
@@ -0,0 +1,233 @@
+    values of a #stat_addon
+    """
+    logger.info("Statting node: {0}".format(src_pk))
+    create_app_context()
+    src = Node.load(src_pk)
+    targets = [src.get_addon(name) for name in settings.ADDONS_ARCHIVABLE]
+    return group(
+        stat_addon.si(
+            addon.config.short_name,
+            src_pk,
+            dst_pk,
+            user_pk,
+        )
+        for addon in targets if addon
+    )()
+
+@celery_app.task(name="archiver.make_copy_request")
+def make_copy_request(dst_pk, url, data):
+    """Make the copy request to the WaterBulter API and handle
+    successful and fauled responses
+
+    :param dst_pk: primary key of registration node
+    :param url: URL to send request to
+    :parama data: <dict> of setting to send in POST to WaterBulter API
+    :return: None
+    """
+    dst = Node.load(dst_pk)
+    provider = data['source']['provider']
+    update_status(dst, provider, ARCHIVER_SENDING)
+    logger.info("Sending copy request for addon: {0} on node: {1}".format(provider, dst_pk))
+    res = requests.post(url, data=json.dumps(data))
+    logger.info("Copy request responded with {2} for addon: {0} on node: {1}".format(provider, dst_pk, res.status_code))
+    update_status(dst, provider, ARCHIVER_SENT)
+    if res.status_code not in (200, 201, 202):
+        catch_archive_addon_error(dst, provider, errors=[res.json()])
+    elif res.status_code in (200, 201):
+        update_status(dst, provider, ARCHIVER_SUCCESS)
+    project_signals.archive_callback.send(dst)
+
+@celery_app.task(name="archiver.archive_addon")
+def archive_addon(addon_short_name, src_pk, dst_pk, user_pk, stat_result):
+    """Archive the contents of an addon by making a copy request to the
+    WaterBulter API
+
+    :param addon_short_name: AddonConfig.short_name of the addon to be archived
+    :param src_pk: primary key of node being registered
+    :param dst_pk: primary key of registration node
+    :param user_pk: primary key of registration initatior
+    :return: None
+    """
+    logger.info("Archiving addon: {1} on node: {0}".format(src_pk, addon_short_name))
+    create_app_context()
+    src = Node.load(src_pk)
+    dst = Node.load(dst_pk)
+    update_status(dst, addon_short_name, ARCHIVER_PENDING, meta={
+        'stat_result': str(stat_result),
+    })
+    user = User.load(user_pk)
+    src_provider = src.get_addon(addon_short_name)
+    folder_name = archive_folder_name(src_provider)
+    provider = src_provider.config.short_name
+    cookie = user.get_or_create_cookie()
+    data = dict(
+        source=dict(
+            cookie=cookie,
+            nid=src_pk,
+            provider=provider,
+            path='/',
+        ),
+        destination=dict(
+            cookie=cookie,
+            nid=dst_pk,
+            provider=ARCHIVE_PROVIDER,
+            path='/',
+        ),
+        rename=folder_name
+    )
+    copy_url = settings.WATERBUTLER_URL + '/ops/copy'
+    make_copy_request.si(dst_pk, copy_url, data)()
```

Comment:
I think given that these are subtasks a synchronous call is ok here -- the archive_addon task should not report success until the make_copy_request subtask is complete. 

An alternative might be to remove the @app.task decorator from make_copy_request and just make that call directly. 

Do you feel strongly about this?